core-api/src/main/java/com/sequenceiq/cloudbreak/validation/externaldatabase/RdsRequestValidator.java

```diff
@@ -23,9 +23,13 @@ public boolean isValid(RDSConfigRequest request, ConstraintValidatorContext cont
 
         Optional<SupportedExternalDatabaseServiceEntry> serviceEntry = SupportedDatabaseProvider.supportedExternalDatabases()
                 .stream()
-                .filter(item -> item.getName().equals(request.getType()) || item.getDisplayName().equals(request.getType()))
+                .filter(item -> item.getName().toLowerCase().equals(request.getType().toLowerCase())
+                        || item.getDisplayName().toLowerCase().equals(request.getType().toLowerCase()))
```

Comment:
In my opinion both L26 and L27 would be more elegant with one single equalsIgnoreCase() instead of double lowerCase() and an equal() call.